test/sales_engine_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/sales_engine'
+
+class SalesEngineTest < Minitest::Test
+  def test_it_exists
+    se = SalesEngine.from_csv({
+                              :items     => "./data/items.csv",
+                              :merchants => "./data/merchants.csv",
+                              })
+
+    assert_instance_of SalesEngine, se
+  end
+
+  def test_it_hasa_readable_attrabutes
+    se = SalesEngine.from_csv({
+                              :items     => "./data/items.csv",
+                              :merchants => "./data/merchants.csv",
```

Comment:
because this was a copy/paste from the iteration 0 instructions, i am going to leave it as a double quote. Unless y'all think I should change it.

Reply:
I think it is fine as is for now. If we really want to, we can change it for the sake of Hound once we are happy with the project in terms of completion. 
I will merge this PR so that we can all get it on our local and hammer out i0. 
But for future reference, I do think we should have at least 3 comments on each PR (not counting the PR author) approving/discussing the PR before we merge, yay or nay? 

💪🏼